File: app/src/main/java/com/innovation4you/napking/app/NapKingApplication.java
package com.innovation4you.napking.app;

import android.app.Application;

import com.innovation4you.napking.data.NapKingService;
import com.innovation4you.napking.data.provider.NapKingJSONDataProvider;

public class NapKingApplication extends Application {

	@Override
	public void onCreate() {
		super.onCreate();

		NapKingService.init(new NapKingJSONDataProvider(this));
	}
}

File: app/src/main/java/com/innovation4you/napking/app/NapKingApplication.kt
package com.innovation4you.napking.app

import android.app.Application

import com.innovation4you.napking.data.NapKingRepository
import com.innovation4you.napking.data.provider.NapKingStaticDataProvider

class NapKingApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        NapKingRepository.init(NapKingStaticDataProvider(this))
    }
}
